packages/truffle-core/lib/commands/debug.js

```diff
@@ -54,6 +54,7 @@ var command = {
 
       var txHash = config._[0];
 
+      var jumpCount = 0;
```

Comment:
So, this PR doesn't seem like a bad idea, but I'm leery of the implementation.  This variable you've added is basically reimplementing the `trace.index` selector, right?  I think it would make more sense to get the info from that existing variable (in this context, it should be accessible as `session.view(trace.index)`) rather than adding a second counter that does essentially the same thing.